### PR TITLE
Added menu option to toggle overmap cursor

### PR DESCRIPTION
**Pull request by ghost**
Added a menu option under interface to toggle overmap cursor. On by default, behaviour doesn't change. If disabled only a blue highlight is shown.

### CODE REVIEW

**Comment by codemime** [comment]
This option would be more useful, if it could be toggled right from the overmap screen. Like it's done for: B - toggle blinking, O - toggle overlays, e.t.c.

**Comment by Nasuga** [comment]
Summary of changes:

- Removed the Option in the Options Menu that toggled OVERMAP_LARGE_CURSOR

- Removed checks for OVERMAP_LARGE_CURSOR

- Created uistate action TOGGLE_CURSOR

- Check for uistate.overmap_cursor is last in the if statement concerning blinking and cursor drawing

File: src/overmap.cpp [hunk 1]
```diff
@@ -2507,6 +2512,9 @@ tripoint overmap::draw_overmap(const tripoint &orig, const draw_data_t &data)
             } else {
                 show_explored = true;
             }
+        } else if (action == "TOGGLE_CURSOR") {
+            uistate.overmap_cursor = !uistate.overmap_cursor;
+
```
Comment (sethsimon) [inline src/overmap.cpp]: Minor: That blank line isn't needed. Besides that, the code looks good, but I'll compile and test it to be sure.

Also, best practice is to keep your master branch up to date with CleverRaven's master branch, and to do PRs on separate feature branches (see the Guidelines section in `CONTRIBUTING.md`).

File: src/uistate.h [hunk 2]
```diff
@@ -56,6 +56,7 @@ class uistatedata : public JsonSerializer, public JsonDeserializer
 
         bool editmap_nsa_viewmode = false;      // true: ignore LOS and lighting
         bool overmap_blinking = true;           // toggles active blinking of overlays.
+        bool overmap_cursor = true;             // toggles overmap cursor
```
Comment (sethsimon) [inline src/uistate.h]: The value of this variable should persist across saves (see `serialize` and `deserialize` in the same file).

**Comment by mugling** [comment]
Appears stalled, closing to reduce PR queue to potentially mergeable requests

Next steps
* Reopen as feature branch
* Fix merge conflicts with `master`
* Complete code review tasks

**Pull request closed without merging**